### PR TITLE
Fixed some imports

### PR DESCRIPTION
**Pull request by anoopshrma**
# Description

- In this doc: https://docs.llamaindex.ai/en/stable/examples/llm/localai.html#llamaindex-interaction `LocalAI_Default` import was done from `llama_index.core.llms` where it is unavailable since it is part of integration LLMs. 
We cannot import it from LocalAI llm too as it requires separate pypi installation for localAI for openAILike.

- Added one util file to put the defaults. 
- Added MockLLM to init file

Fixes # (issue)

## Type of Change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?
- [x] I stared at the code and made sure it makes sense

### CODE REVIEW

File: llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+from types import MappingProxyType
+from typing import Any, Dict
+
+# Use these as kwargs for OpenAILike to connect to LocalAIs
+DEFAULT_LOCALAI_PORT = 8080
```
Comment (anoopshrma) [inline llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: Want to change this to OPENAI_LIKE_DEFAULTS but ideally these are LocalAI defaults

confused on this 🤯 , any suggestion would be great!

Comment (anoopshrma) [reply llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: I'm thinking can we have a file for constants in `llama-index-integrations`  to put constants like the above one for easy access in all the third party LLMs?

Comment (logan-markewich) [reply llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: I think tbh localai is deprecated in favor of openai-like. I can't remember if the localai package still exists or if it was aliased to openai-like lol

Comment (anoopshrma) [reply llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: The package exist for LocalAI: https://pypi.org/project/llama-index-llms-localai/
So the doc should also be updated:https://docs.llamaindex.ai/en/stable/examples/llm/localai.html#llamaindex-interaction 
as it points towards LocalAI but uses OpenAILike 😄  class to interaction.

Comment (anoopshrma) [reply llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: What should be the course of action on this @logan-markewich  then if localAI is going to be deprecated!.

Comment (logan-markewich) [reply llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: Yeaaa I think localai mistakingly got made into a package lol. I saw we deal with it in another PR 😅

Comment (anoopshrma) [reply llama-index-integrations/llms/llama-index-llms-openai-like/llama_index/llms/openai_like/utils.py:5]: Ah okay, so if you look at the changes I made in this PR is actually removing dependency from localai such as importing default values from localai llm package

**Pull request closed without merging**